controller/workitemtype_blackbox_test.go

```diff
@@ -93,6 +95,20 @@ var (
 	personID = uuid.FromStringOrNil("22a1e4f1-7e9d-4ce8-ac87-fe7c79356b16")
 )
 
+func getCategory(db application.DB) uuid.UUID {
```

Comment:
What is the purpose of this function? Why do you just take the first category and not load one explicitly?